src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Views.ts

```diff
@@ -640,6 +642,69 @@ export var adhUserActivityOverviewDirective = (
     };
 };
 
+export var adhUserProfileImageDirective = (
+    adhHttp: AdhHttp.Service<any>,
+    adhConfig: AdhConfig.IService
+) => {
+    return {
+        restrict: "E",
+        scope: {
+            path: "@",
+            format: "@?", // thumbnail [default] or detail
+            isImageMissing: "=?",
+            cssClass: "@?"
+        },
+        templateUrl: adhConfig.pkg_path + pkgLocation + "/UserProfileImage.html",
+        link: (scope) => {
+            scope.config = adhConfig;
+            scope.isImageMissing = false;
+            var handleImageMissing = () => scope.isImageMissing = true;
+            scope.$watch("path", (path) => {
+                if ( ! path) { return; }
+
+                adhHttp.get(scope.path).then((user) => {
+                    scope.assetPath = user.data[SIImageReference.nick].picture;
+                    scope.userName = user.data[SIUserBasic.nick].name;
+                    if ( ! scope.assetPath) {
+                        handleImageMissing();
+                    }
+               },
+               handleImageMissing
+               );
```

Comment:
This indentation looks off. First of all, there is a space missing for the last 3 lines. And I would also combine them into one: `}, handleImageMissing);`